web/source/osk/browser/keytip.ts

```diff
@@ -130,7 +137,7 @@ namespace com.keyman.osk.browser {
       h2 = h2 > hBoundedMax ? hBoundedMax : h2;
       h3 = hMax > hBoundedMax ? hBoundedMax : h3;
 
-      if(device.OS == 'Android') {
+      if(device.OS == 'android') {
```

Comment:
Were these lines just wrong before?

Reply:
This is due to the change in type for `device`.  The "core-spec" version uses all lower case, in contrast to KMW's original `Device`-class's implementation.